<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaPrincipalCliente</class>
 <widget class="QMainWindow" name="ventanaPrincipalCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="TTelefono">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BCancelar">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TCorreo">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TCedulaR">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TNombre">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LTelefono">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Telefono</string>
      </property>
     </widget>
     <widget class="QLabel" name="LIngresar">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese su numero de cedula para ingresar</string>
      </property>
     </widget>
     <widget class="QLabel" name="LBienvenida">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bienvenido al supermercado</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCorreo">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Correo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BIngresar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingresar</string>
      </property>
     </widget>
     <widget class="QLabel" name="LNoReg">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Usted no se encuentra registrado</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCedulaR">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cedula</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CBCiudad">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LPregunta">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>¿Desea registrarse?</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCiudad">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ciudad</string>
      </property>
     </widget>
     <widget class="QLabel" name="LNombre">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BRegistrar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="LDatos">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese sus datos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BSi">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>SÍ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BNo">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>NO</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TCedula">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LCedula">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cedula</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
